spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/DefaultAccessTokenConverter.java

```diff
@@ -55,14 +55,32 @@ public void setUserTokenConverter(UserAuthenticationConverter userTokenConverter
 	}
 
 	/**
-	 * Flag to indicate the the grant type should be included in the converted token.
+	 * Converter for the part of the data in the token representing a user.
+	 *
+	 * @return the set userTokenConverter
+	 */
+	public UserAuthenticationConverter getUserTokenConverter() {
```

Comment:
I don't feel this getter is required. Is there a specific use case where you need access to this?

Reply:
The getters were added so that child classes could access the private members (userTokenConverter, includeGrantType, scopeAttribute, clientIdAttribute) from overriden methods such as convertAccessToken and extractAuthentication.  Changing the scope of these new getters to protected or changing the fields from private to protected would also work.